libs/features/placeholders.js

```diff
@@ -1,30 +1,94 @@
-const placeholders = {};
-let fetched = false;
+const cachedPlaceholders = {};
 
 // eslint-disable-next-line no-async-promise-executor
-const fetchPlaceholders = (config) => new Promise(async (resolve) => {
-  if (fetched) { resolve(placeholders); return; }
+const fetchPlaceholders = (config, sheet) => new Promise(async (resolve) => {
+  if (cachedPlaceholders?.[config.locale.ietf]?.[sheet]) {
+    resolve(cachedPlaceholders[config.locale.ietf][sheet]);
+    return;
+  }
+
   const path = `${config.locale.contentRoot}/placeholders.json`;
-  const resp = await fetch(path);
+  const query = sheet !== 'default' && typeof sheet === 'string' && sheet.length ? `?sheet=${sheet}` : '';
+  const resp = await fetch(`${path}${query}`);
   const json = resp.ok ? await resp.json() : { data: [] };
-  if (json.data.length === 0) { resolve(placeholders); return; }
+  if (json.data.length === 0) { resolve({}); return; }
+  cachedPlaceholders[config.locale.ietf] = cachedPlaceholders[config.locale.ietf] || {};
+  cachedPlaceholders[config.locale.ietf][sheet] = {};
   json.data.forEach((item) => {
-    placeholders[item.key] = item.value;
+    cachedPlaceholders[config.locale.ietf][sheet][item.key] = item.value;
   });
-  fetched = true;
-  resolve(placeholders);
+  resolve(cachedPlaceholders[config.locale.ietf][sheet]);
 });
 
-function findPlaceholder(suppliedPlaceholders, key) {
-  return suppliedPlaceholders[key] || key.replaceAll('-', ' ');
+function keyToStr(key) {
+  return key.replaceAll('-', ' ');
 }
 
-export async function replaceText(config, regex, text) {
-  const fetchedPlaceholders = await fetchPlaceholders(config);
-  return text.replaceAll(regex, (_, key) => findPlaceholder(fetchedPlaceholders, key));
+async function getPlaceholder(key, config, sheet) {
+  const defaultLocale = 'en-US';
+
+  const getDefaultPlaceholders = async () => {
+    // Maybe we should modify the getConfig() method to provide a default config?
+    // Could contentRoot be different than the origin?
+    const defaultConfig = {
+      locale: {
+        ietf: defaultLocale,
+        contentRoot: window.location.origin,
+      },
+    };
+
+    const defaultPlaceholders = await fetchPlaceholders(defaultConfig, sheet)
```

Comment:
In theory I'd agree with that, but historical evidence is that things are just forgotten often for non-English locales which is something we encountered often enough in production in the dexter/aem world.

It's a large number of pages and content so IMO it's better if the user sees something understandable. (Assuming english is more understandable )

Reply:
If we assume a content producer is able to spot an English key name on a German locale, as is the current case with missing placeholders, I think it's safe to assume they would also be able to spot an English word/sentence on a German page. So if content producers pay attention to these details, we have nothing to worry about. However, if this detail is missed, then at least the end user gets some relevant information, albeit in another (international) language. In the future, we were thinking to include some LANA logging when placeholders use fallbacks so we see where translations are missing.

Reply:
I'm not sure I agree that users have the same chance of spotting keys vs translated text: 
Here's are two example pages I made. Open the first link and try to see the English text. Then open the second link: 
https://main--milo--adobecom.hlx.page/drafts/vhargrave/normaltext
https://main--milo--adobecom.hlx.page/drafts/vhargrave/keytext

Maybe we can extend this in a later task to still show the keys on stage, but show the backup text on prod? That way we get the best of both worlds. What do you guys think?

Reply:
Instead of showing different content on STAGE vs PROD, why don't we just highlight backup text when a given query param or global object is set? We would be able to use it in our automated tests and output all issues. @overmyheadandbody's suggestion with LANA logging is another great way of capturing issues and fixing them asap.

Reply:
That's a great idea .
And yeah, I forgot to write it, but the Lana logging would also be a super addition.

Reply:
Although that particular example from Victor's test tries to prove a point, I'd argue that the effect is due to the brackets and dashes. This is not how things behave today in Milo.

Today, if a label is not found for the `you-saw-this-quicker-right` label, it gets replaced with `you saw this quicker right`, not with `[[you-saw-this-quicker-right]]` as that page might suggest. So it's exactly the same when trying to spot a missed translation.

Reply:
Yes, that's true. My dream scenario would also include not removing the brackets from the placeholder, so they stick out. 
Technically fine for me regarding your proposal, but one of the admins (chris m., ryan, chris p.) has to merge, since they're admins.